src/services/services.inteface.ts

```diff
@@ -0,0 +1,48 @@
+export interface RequestProps {
```

Comment:
The filename has a typo: services.*inteface*.ts

Also, I believe we should declare interfaces where they are used. But we can debate that at the next meeting.